osm2lanes/src/test.rs

```diff
@@ -122,14 +129,11 @@ impl std::fmt::Display for TestCase {
 }
 
 /// Get Test Cases from tests.yml
+#[must_use]
 pub fn get_tests() -> Vec<TestCase> {
-    let tests: Vec<TestCase> =
-        serde_yaml::from_reader(BufReader::new(File::open("../data/tests.yml").unwrap()))
-            .expect("invalid yaml in data/tests.yml");
-    let tests: Vec<TestCase> = tests
-        .into_iter()
-        .filter(|test| test.test_enabled())
-        .collect();
+    let tests: Vec<TestCase> = serde_yaml::from_str(include_str!("../../data/tests.yml"))
+        .expect("invalid yaml in data/tests.yml");
+    let tests: Vec<TestCase> = tests.into_iter().filter(TestCase::test_enabled).collect();
```

Comment:
we now include the tests at compile time (File::open not supported in wasm)

downside: every test change triggers a recompile.
necessary evil?

I could make a crate dedicated to this, or have platform specific implementation, but both seem worse.

Reply:
As long as this is feature-gated, seems fine to me. Test changes triggering a recompile really is necessary for the web app.

FWIW, there are even crates like https://github.com/Michael-F-Bryan/include_dir/ to shove more stuff into WASM at compile-time. I... have definitely included about 500MB of binary files before. :D